packages/trader/src/Modules/SmartChart/Components/all-markers.jsx

```diff
@@ -1,9 +1,10 @@
+/* eslint-disable no-console */
```

Comment:
Just for my understanding what is the reason behind using this? is there any console.log inside this file? 
Can we remove it?

Reply:
We can remove. There's no console.log in this file. Must have been left over when we were debugging.

Reply:
good catch! I've removed it)

Reply:
Thanks @maryia-deriv